### PR TITLE
Fix window.open() looks not working

### PR DESCRIPTION
**Pull request by yuya-oc**
Before submitting, please confirm you've
 - [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [x] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
Fix `window.open()` function looks not working.

When a new window was opened, it was hidden because the main window's `{show:false}` option was inherited to the new window. So the new window was not visible.

**Issue link**
#450 

**Test Cases**
1. Click "Help" below message box.
2. A new window show appear.

**Additional Notes**
https://circleci.com/gh/yuya-oc/desktop/168#artifacts

### CODE REVIEW

**Comment by jasonblais** [comment]
Testing now, on Windows 10.

File: src/browser/components/MattermostView.jsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const MattermostView = React.createClass({
       }
       if (currentURL.host === destURL.host) {
         // New window should disable nodeIntergration.
-        window.open(e.url, 'Mattermost', 'nodeIntegration=no');
+        window.open(e.url, 'Mattermost', 'nodeIntegration=no,show=yes');
```
Comment (jasonblais) [inline src/browser/components/MattermostView.jsx]: should there be a space between the comma and `show`

**Comment by jasonblais** [comment]
This works on Windows, thanks for the quick fix!

**Comment by yuya-oc** [comment]
Thanks for testing!

**Pull request merged**